testing/src/cypress/e2e/04-api-institution/01-instition-dashboard.cy.ts

```diff
@@ -0,0 +1,150 @@
+const COLLEGE_F = Cypress.env("COLL_F");
+const USER_F = Cypress.env("USERS").coll_f;
```

Comment:
for my info. what is the difference between `COLLEGE_F` and `USER_F`?

Reply:
College_F is the institution centric and user_f is user centric (like the login access and user details etc). For now we do have both of the identities same. But if they do change, we need to be able to accommodate

Reply:
Will update the variable names to be generic, and the env will be representation of the scenario.

Reply:
Done